plugins/inputs/ldap_response/ldap_response.go

```diff
@@ -0,0 +1,209 @@
+package ldap_response
+
+import (
+	ctls "crypto/tls"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"gopkg.in/ldap.v2"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Ldap struct {
+	Host               string
+	Port               int
+	Ssl                string
+	InsecureSkipVerify bool
+	SslCa              string
+	BindDn             string
+	BindPassword       string
+	SearchBase         string
+	SearchFilter       string
+	SearchAttributes   []string
+}
+
+const sampleConfig string = `
+  host = "localhost"
+  port = 389
+
+  # ldaps, starttls, or no encryption. default is an empty string, disabling all encryption.
+  # note that port will likely need to be changed to 636 for ldaps
+  # valid options: "" | "starttls" | "ldaps"
+  ssl = ""
+
+  # skip peer certificate verification. Default is false.
+  insecure_skip_verify = false
+
```

Comment:
Please comment all options that are holding the default and don't need to be touched by the user by default. Furthermore, comments should start with a double-hash.
```suggestion
  # host = "localhost"
  # port = 389

  ## ldaps, starttls, or no encryption. default is an empty string, disabling all encryption.
  ## note that port will likely need to be changed to 636 for ldaps
  ## valid options: "" | "starttls" | "ldaps"
  # ssl = ""

  ## skip peer certificate verification. Default is false.
  ## insecure_skip_verify = false

```
Not sure about the `host` and `port`, but you should only leave options uncommented that _have_ to be set and do not have a sensible default.

Same for all other lines below.